src/sync/pull-request.ts

```diff
@@ -66,7 +68,12 @@ export const getPullRequestTask = async (
 				direction: SortDirection.DES
 			});
 
-	statsd.timing(metricHttpRequest.syncPullRequest, Date.now() - startTime, 1, [`status:${status}`]);
+	const gitHubVersion = getCloudOrServerFromHost(request.host);
+	statsd.timing(
+		metricHttpRequest.syncPullRequest,
+		Date.now() - startTime,
+		1,
```

Comment:
/nit /notSureIfICare Is it worth adding a comment beside `,1` like `Sample rate is set to 100%`